src/platform.cpp

```diff
@@ -254,6 +254,21 @@ internal void DebugHandleKeyEvent(SDL_KeyboardEvent Event, sdl_setup *Setup)
             bool IsFullscreen = SDL_GetWindowFlags(Setup->Window) & FullscreenFlag;
             SDL_SetWindowFullscreen(Setup->Window, IsFullscreen ? 0 : FullscreenFlag);
         }
+        else if (IsDown && KeyCode == SDLK_l)
+        {
+            ++Recording->ActionIndex;
+            if (Recording->ActionIndex > 2)
+            {
+                Recording->ActionIndex = 0;
+                // Note(Karsten): Need a more structured way to detect reset of looped input, so we can reset keyboard.
```

Comment:
Should probably set some sort of cleanup flag here that can be read and dealt with in Game loop when checking for record/playback ActionIndex.